src/ServiceControl.Config/Commands/StartServiceInMaintenanceModeCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace ServiceControl.Config.Commands
+{
+    using System;
+    using System.Threading.Tasks;
+    using Caliburn.Micro;
+    using ServiceControl.Config.Events;
+    using ServiceControl.Config.Framework;
+    using ServiceControl.Config.Framework.Commands;
+    using ServiceControl.Config.UI.AdvanceOptions;
+    using ServiceControlInstaller.Engine.ReportCard;
+
+    class StartServiceInMaintenanceModeCommand : AwaitableAbstractCommand<AdvanceOptionsViewModel>
+    {
+        private readonly IWindowManagerEx windowManager;
+        private readonly IEventAggregator eventAggregator;
```

Comment:
I believe it does.  We can standardize but lets do that as a separate PR / refactoring exercise